src/sensor/sensor.cpp

```diff
@@ -105,4 +108,60 @@ void Sensor::connectLinkLog(const LinkConfiguration& logConf)
     emit linkLogUpdate();
 }
 
+void Sensor::setControlPanel(const QUrl& url)
+{
+    if(!url.isValid()) {
+        qCWarning(PING_PROTOCOL_SENSOR) << "Invalid url:" << url;
```

Comment:
What about using Critical if these are invalid? You are unable to use the sensor properly without them, right?